src/components/common/CommitInfoDialog.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import { withTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormLabel,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import { formatDistance } from 'date-fns';
+import { enGB, fr } from 'date-fns/locale';
+import { closeCommitInfoDialog } from '../../actions';
+import { CODE } from '../../config/appInstanceResourceTypes';
+import {
+  DEFAULT_INSTRUCTOR_COMMIT_INFO,
+  PLACEHOLDER_DATE,
+} from '../../config/settings';
+
+// to add a new language to the dates
+const locales = { fr, en: enGB };
```

Comment:
It think you can import this from `i18n`.

Reply:
Did you see if this was possible @spaenleh ?

Reply:
@juancarlosfarah I do this to map our languages to the `Locale` type defined by `date-fns`. I don't see how i would use `i18n` in this context.

Reply:
@spaenleh, no worries. Not important. We can discuss this next week.